src/_sass/_base.scss

```diff
@@ -58,10 +71,17 @@ h3 {
   font-size: calc(24rem / 16);
 }
 
-h4 {
+h4,
+.h4 {
   font-size: calc(16rem / 16);
 }
 
+.italics {
```

Comment:
@segacy1 I've added a style called `italics` that represents the italicized, `#136c97`, 14px size font. This is not on the Figma Style Guide/Type section. https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=3725%3A10998&t=dlLp11tKHkYYCsyF-0

Reply:
Hi @machikoyasuda the blue color was an error! We can just make that our primary blue. I'll update in figma. 

For the italics - it's the "subtext" style italicized, so I considered it more of an override than its own style. I think calling italics works atm!

Reply:
Glad I asked :) Thank you for clarifying and responding quickly!